ui/src/pages/Upload.tsx

```diff
@@ -0,0 +1,261 @@
+            rows={8}
+            required
+            id="story-summary"
+            name="summary"
+            label="Story Summary"
+          />
+          <TextField
+            onChange={handleChange}
+            variant="outlined"
+            required
+            id="author-first-name"
+            name="author_first_name"
+            label="First Name"
+            placeholder="Lorem ipsum"
+          />
+          <TextField
+            onChange={handleChange}
+            variant="outlined"
+            required
+            id="author-last-name"
+            name="author_last_name"
+            label="Last Name"
+            placeholder="Lorem ipsum"
+          />
+          <TextField
+            onChange={handleChange}
+            multiline
+            placeholder="Enter additional information here"
+            rows={8}
+            id="author-bio"
+            name="bio"
+            label="Author Bio"
+          />
+
+          <FormControl>
+            <InputLabel id="Country of Origin">Country of Origin</InputLabel>
+            <Select
+              value={formInput.author_country}
+              onChange={handleChange}
+              inputProps={{
+                name: "author_country",
+                id: "select-label-country",
+              }}
+            >
+              {countriesList.map((country) => (
+                <MenuItem key={country.code} value={country.name}>
+                  {country.name}
+                </MenuItem>
+              ))}
+            </Select>
+          </FormControl>
+
+          <FormControl>
+            <InputLabel id="Current Location">Current City</InputLabel>
+            <Select
+              value={formInput.current_city}
+              onChange={handleChange}
+              inputProps={{
+                name: "current_city",
+                id: "select-label-city",
+              }}
+            >
+              <MenuItem value={"Toronto"}>Toronto</MenuItem>
+              <MenuItem value={"Calgary"}>Calgary</MenuItem>
+              <MenuItem value={"Vancouver"}>Vancouver</MenuItem>
+              <MenuItem value={"Thunder Bay"}>Thunder Bay</MenuItem>
+            </Select>
+          </FormControl>
+
+          <FormControl>
+            <InputLabel id="input-label-year">Year Published</InputLabel>
+            <Select
+              value={formInput.year}
+              onChange={handleChange}
+              inputProps={{
+                name: "year",
+                id: "select-label-year",
+              }}
+            >
+              <MenuItem value={"2021"}>2021</MenuItem>
+              <MenuItem value={"2020"}>2020</MenuItem>
+              <MenuItem value={"2019"}>2019</MenuItem>
+            </Select>
+          </FormControl>
+
+          <DropzoneArea
+            acceptedFiles={["image/*"]}
+            dropzoneText={"Drag and drop an image here or click"}
+            onChange={(files) => {
+              setImage(files);
+            }}
```

Comment:
Let's set a filesLimit to make sure only 1 image is uploaded.
```suggestion
          <DropzoneArea
            acceptedFiles={["image/*"]}
            filesLimit={1}
            dropzoneText={"Drag and drop an image here or click"}
            onChange={(files) => {
              setImage(files);
            }}
```